tasks/README.md

```diff
@@ -1,6 +1,82 @@
 # gem5art tasks package
 
-This package contains a [Celery](http://www.celeryproject.org/) worker and tasks for running gem5.
+This package contains two parallel task libraries for running gem5 experiments.
```

Comment:
So, just to make sure: we are going to keep this documentation both on the website and in the gem5art source as well?

Reply:
That's my thought. We'll just have the */README.md in the source and all of the docs/ on the gem5 website.